gutenberg-starter-theme-blocks/functions.php

```diff
@@ -22,24 +22,24 @@ function gutenberg_starter_theme_blocks_support()  {
 		add_theme_support( 'editor-color-palette', array(
```

Comment:
When I customize colors (e.g. make the paragraph text color Strong Blue), they aren't being applied in the editor right. They do show up in the front-end though.

Reply:
That's odd — I wonder if there's some sort of conflict between the `add_theme_support` colors and the ones in `experimental-theme.json`.

Reply:
Hey would you mind re-testing this? I'm not able to reproduce.